operations/use-mysql-version-8.0.yml

```diff
@@ -0,0 +1,4 @@
+---
+- type: replace
+  path: /instance_groups/name=database/jobs/name=pxc-mysql/properties/mysql_version?
+  value: "8.0"
```

Comment:
We typically add new ops-files to the `experimental` section to give us a chance to validate them in our CI before "promoting" them to main "stable" set of ops-files.

Could you please move this file (and the associated test and description) accordingly? It would also be helpful if you could help us identify which scenarios we should test in CI. For example, should we be validating an upgrade from PXC 5.7 to 8.0?

Reply:
You should definitely test an upgrade path from 5.7 to 8.0. We have pretty extensive CI tests around upgrades. We deploy a new CF with pxc 8.0. We don't upgrade a CF from 5.7 to 8.0.